tests/src/Traits/PathAccessTrait.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase\Traits;
+
+/**
+ * Provides methods to check access to paths.
+ */
+trait PathAccessTrait {
```

Comment:
This trait contains path assertions, let's rename it accordingly:

```suggestion
trait AssertPathAccessTrait {
```